tests/certification/pubsub/pulsar/pulsar_test.go

```diff
@@ -190,6 +192,17 @@ func TestPulsar(t *testing.T) {
 			dockerComposeYAML: filepath.Join(dir, "docker-compose.yaml"),
 			componentsPath:    filepath.Join(dir, "components/auth-oauth2"),
 			services:          []string{"zookeeper", "pulsar-init", "bookie", "broker"},
+			tlsEnabled:        false,
+		})
+	})
+
+	t.Run("TLS", func(t *testing.T) {
+		suite.Run(t, &pulsarSuite{
+			authType:          "none",
+			dockerComposeYAML: dockerComposeTLSYAML,
+			componentsPath:    "./components/tls",
```

Comment:
You have six very differently configured pulsar components loaded here. How do you verify that ALL of them actually worked?

And if your server (docker file) is configured to require a certificate, then you really shouldn't be able to connect using an insecure mode like in `consumer_four` for example.

Reply:
All of the configs have `enableTSL: true`. They are connecting over a TLS connection to send and receive messages over a TLS connection but allow us to skip Hostname validation on the key and setting `tlsAllowInsecureConnection: true` since the certificate isn't signed by an approved CA.